ci-operator/config/openshift/cluster-api-operator/openshift-cluster-api-operator-main.yaml

```diff
@@ -43,10 +43,50 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-aws
+- as: e2e-aws-ovn
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: e2e-aws-ovn-serial
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-serial
+- as: e2e-aws-ovn-techpreview
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-capi
+- as: e2e-aws-capi-techpreview
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      commands: make e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: openshift-e2e-aws-capi
+- as: e2e-gcp-ovn-techpreview
+  steps:
+    cluster_profile: gcp
+    workflow: openshift-e2e-gcp-capi
+- as: e2e-gcp-capi-techpreview
+  steps:
+    cluster_profile: gcp
+    test:
+    - as: test
+      commands: make e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: openshift-e2e-gcp-capi
+- as: e2e-azure-ovn-techpreview
+  steps:
+    cluster_profile: azure4
+    workflow: openshift-e2e-azure-capi
```

Comment:
Does Azure work yet?

Reply:
No, it doesn't work yet, but I hope we will enable it soon (when custom backend pool names PR is merged).